dashboard/lib/acapela.rb

```diff
@@ -24,6 +25,18 @@ def acapela_text_to_audio_url(text, voice="rosie22k", speed=180, shape=100)
   request = VAAS_HASH.merge(params)
   response = Net::HTTP.post_form(URI.parse(VAAS_URL), request)
 
+  metrics = [
+    {
+      metric_name: :AcapelaAPICall,
+      dimensions: [
+        {name: "Environment", value: CDO.rack_env},
+        {name: "Context", value: context}
```

Comment:
might be useful to track voice here, too; we use a different voice for each language

Reply:
Agree. I tried that before but hit complications from the CloudWatch side. CloudWatch treats [each metric](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_concepts.html#Metric) as an unique combination of namespace, metric name and dimension values. Thus, adding a dimension means splitting a metric into several smaller metrics. We then have to use [Search expression and Metric math](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/search-expression-syntax.html) to aggregate them. 

However, this is something worth trying since the lesson learnt will be useful to other monitoring problems later.

Reply:
hah, that's much more complicated than I thought in would be! In that case I absolutely agree that these two dimensions seem sufficient

Reply:
I already added the 3rd dimension for voice and created a follow-up task https://codedotorg.atlassian.net/browse/FND-1464 to aggregate data in CloudWatch. This will be an interesting learning experience :)

Reply:
nice!